src/Symfony/Component/DependencyInjection/Config/AutowireServiceResource.php

```diff
@@ -34,12 +34,13 @@ public function __construct($class, $path, array $autowiringMetadata)
 
     public function isFresh($timestamp)
     {
-        if (!file_exists($this->filePath)) {
+        $filemtime = @filemtime($this->filePath);
+        if (!$filemtime) {
```

Comment:
The changes in this file can probably also be applied on a lower branch.

Reply:
same as @sroze you can inline all this condition